README.md

```diff
@@ -10,9 +10,10 @@ The following must be done before running the Flask App.
 
 Create a text file, `.env`, that must have the following:
 ```
-TRELLO_KEY=<Trello API key>
-TRELLO_TOKEN=<Trello API token>
-TRELLO_BOARD_ID=<Trello board id>
+USER_NAME=<Mongo User Id>
+PASSWORD=<Mongo password>
+MONGO_URL=<Mongo DB Url>
+DEFAULT_DATABASE=<Mongo DB Name>
 ``` 
```

Comment:
Your app actually requires specific setup of the database too, which needs to be explained in your README - it needs to have a `todo_lists` collection with some specific lists in it. Or even nicer - your app could initialise that collection on startup if it sees it is empty.

And just below here, you still discuss Trello instead.